### PR TITLE
updating docs

### PR DESCRIPTION
**Pull request by nadijagraca**
## Description
Made it clearer that the build function needs to be run at the end, add full example to the code cell. 

## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

  - I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
  - I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorized to submit this contribution on behalf of the original creator(s) or their licensees.
  - I certify that the use of this contribution as authorized by the Apache 2.0 license does not violate the intellectual property rights of anyone else.
  - I have not referenced individuals, products or companies in any commits, directly or indirectly.
  - I have not added data or restricted code in any commits, directly or indirectly.

### CODE REVIEW

File: vizro-ai/docs/pages/tutorials/quickstart-dashboard.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ Vizro-AI can generate a multi-page dashboard that includes the following feature
 
 ```text
 user_question = """
-Create a page showing 1 card, 1 chart, and 1 filter.
+Create a page showing 1 card, 1 chart.
```
Comment (stichbury) [inline vizro-ai/docs/pages/tutorials/quickstart-dashboard.md:74]:
```suggestion
Create a page showing 1 card and 1 chart.
```

File: vizro-ai/docs/pages/tutorials/quickstart-dashboard.md [hunk 2]
```diff
@@ -88,12 +88,37 @@ Next, submit the data and prompt string:
 dashboard = vizro_ai._dashboard([df], user_question)
 ```
 
-The call to `_dashboard()` triggers the dashboard building process. Once Vizro-AI finishes the dashboard generation process, you can launch the dashboard with `build()`.
+The call to `_dashboard()` triggers the dashboard building process.
+
+## 7. Build dashboard
+Once Vizro-AI finishes the dashboard generation process, launch the dashboard with `build()`.
+
```
Comment (stichbury) [inline vizro-ai/docs/pages/tutorials/quickstart-dashboard.md:95]: This section is confusing me -- sorry. 

In line 91, we have a call to `_dashboard()` which triggers the "building process".

Then line 94 says "Once Vizro-AI finishes the dashboard generation process" and that you need to call `build()`. 

But didn't line 91 kick off the build according to the text. What's the difference between generation and build?

The names seem odd if the one that doesn't include the word "build" does the building and the one that does have "build" is the launcher. 

Could you clarify for me please @nadijagraca ?

File: vizro-ai/docs/pages/tutorials/quickstart-dashboard.md [hunk 3]
```diff
@@ -88,12 +88,37 @@ Next, submit the data and prompt string:
 dashboard = vizro_ai._dashboard([df], user_question)
 ```
 
-The call to `_dashboard()` triggers the dashboard building process. Once Vizro-AI finishes the dashboard generation process, you can launch the dashboard with `build()`.
+The call to `_dashboard()` triggers the dashboard building process.
```
Comment (lingyielia) [inline vizro-ai/docs/pages/tutorials/quickstart-dashboard.md:91]:
```suggestion
The call to `_dashboard()` triggers the dashboard generation process. By default, it generates the Vizro `Dashobard` Object.
```

File: vizro-ai/docs/pages/tutorials/quickstart-dashboard.md (hunk 2 above)
Comment (lingyielia) [reply vizro-ai/docs/pages/tutorials/quickstart-dashboard.md:95]: Thanks for pointing out, very good point! So to be more specific, 

`_dashboard()` generates the Vizro Dashboard object.

`Vizro().build(dashboard)` builds the dashboard.

**Review by lingyielia — APPROVED** [review]

File: vizro-ai/docs/pages/tutorials/quickstart-dashboard.md (hunk 2 above)
Comment (nadijagraca) [reply vizro-ai/docs/pages/tutorials/quickstart-dashboard.md:95]: As Lingyi mentioned below, `_dashboard()` generates the Vizro Dashboard object, but we need to do `Vizro().build(dashboard).run()` to run the dashboard.

File: vizro-ai/docs/pages/tutorials/quickstart-dashboard.md [hunk 4]
```diff
@@ -88,12 +88,37 @@ Next, submit the data and prompt string:
 dashboard = vizro_ai._dashboard([df], user_question)
 ```
 
-The call to `_dashboard()` triggers the dashboard building process. Once Vizro-AI finishes the dashboard generation process, you can launch the dashboard with `build()`.
+The call to `_dashboard()` triggers the dashboard generation process. By default, it generates the Vizro `Dashobard` Object.
+
+## 7. Build dashboard
+Once Vizro-AI finishes the dashboard generation process, launch the dashboard with `build()`.
```
Comment (stichbury) [inline vizro-ai/docs/pages/tutorials/quickstart-dashboard.md:94]:
```suggestion
The call to `_dashboard()` initiates dashboard generation. By default, it generates the Vizro `Dashobard` Object.

## 7. Build dashboard
Once dashboard generation is complete, launch the dashboard with `build()`.
```

Comment (stichbury) [reply vizro-ai/docs/pages/tutorials/quickstart-dashboard.md:94]: Maybe slightly shorter but what you have is also good.

**Review by stichbury — APPROVED** [review]
LGTM, thanks!

**Pull request merged**